config/deploy/server_settings.rb

```diff
@@ -1,31 +1,9 @@
 # Note: this file is required in config/deploy/<environment>.rb
 
 puts "ENV['APP_ENV']  = #{ENV['APP_ENV']}"
-puts "ENV['SDR_APP']  = #{ENV['SDR_APP']}"
-puts "ENV['SDR_USER'] = #{ENV['SDR_USER']}"
-puts "ENV['SDR_HOST'] = #{ENV['SDR_HOST']}"
-puts "ENV['SDR_URL']  = #{ENV['SDR_URL']}"
 puts
 
-set :default_env, {
-    'APP_ENV'  => ENV['APP_ENV'],
-    'SDR_APP'  => ENV['SDR_APP'],
-    'SDR_USER' => ENV['SDR_USER'],
-    'SDR_HOST' => ENV['SDR_HOST'],
-    'SDR_URL'  => ENV['SDR_URL'],
-}
```

Comment:
Might need to leave the `APP_ENV` in there.  Need to check all the code (not just configs) for any use of these ENV values.  If the use of these values is anywhere in the code, a switch to using rails-config gem and Settings would be good.